### PR TITLE
Fix for issue #939 ansi colors don't work on Windows Terminal

### PR DESCRIPTION
**Pull request by julie777**
## Pull Request Check List

- [ ] A news fragment is added in `news/` describing what is new.
I don't know how to do this
- [ ] Test cases added for changed code.
Not required

## Describe what you have changed in this PR.

I found that a call to os.system("") with no command will set python on Windows to correctly support ANSI output.
ref: https://bugs.python.org/issue40134

### CODE REVIEW

File: pdm/__init__.py [hunk 1]
```diff
@@ -17,6 +18,8 @@
     "Core",
 )
 
+os.system("")
```
Comment (pawamoy) [inline pdm/__init__.py:21]: It would probably be good to add a link to the Python issue in a comment above that statement :slightly_smiling_face:

**Review by frostming — CHANGES_REQUESTED** [review]
Again, the original issue https://bugs.python.org/issue40134 only occurs on windows cmd and Windows Terminal works fine. This fix doesn't make sense.

There should be some issue with your terminal setup.

File: pdm/__init__.py (hunk 1 above)
Comment (julie777) [reply pdm/__init__.py:21]: will do

**Comment by frostming** [comment]
@julie777 ping, Are you still working on this? I hope you understand my concern behind those review comments.

**Comment by frostming** [comment]
This issue should be fixed after the UI framework is switched to rich. Close it for now.

**Pull request closed without merging**